### ISSUE TITLE
Publish to Maven Central

### ISSUE THREAD

**Issue by EronWright** [nupic-community/flink-htm#18]
Publish the first release of the library to Central.

**Comment by EronWright** [comment]
@rhyolight do you have any suggestions about this?   As as nupic-community project are there any special procedures to follow?

**Comment by rhyolight** [comment]
No, not that I know of. You might ask @cogmission, he works with maven central with HTM.Java. I assume you'd follow the same procedure? Honestly I haven't done Java work in so long... the only thing I remember about Maven is constantly misreading `pom.xml` as `porn.xml`.
